docs/feature-flagging/targeting.md

```diff
@@ -40,8 +40,9 @@ import eppo_client
 
 client = eppo_client.get_instance()
 variation = client.get_string_assignment(
-  "<SUBJECT-KEY>", 
-  "<EXPERIMENT-KEY>", 
+  "<SUBJECT-KEY>",
+  "<FLAG-KEY>",
+  "<DEFAULT-VARIATION>",
```

Comment:
This is slightly confusing perhaps: I think it's clearer to say "default value" instead of default variation